### ISSUE TITLE
Using shiny.router with separate ui and server files

### ISSUE THREAD

**Issue by em-dataIntegrityEnthusiast** [Appsilon/shiny.router#80]
Is it possible to use shiny.router with separate ui and server files? I could not figure out how you would get the scoping right on that. Defining the router object in UI would make it not accessible in server and global.R did not seem to help that either.

**Comment by rstammer** [comment]
Yes, in general that is possible! [Here is an older example](https://github.com/rstammer/shiny.blueprint) that follows a little bit your idea, if I got it right. The example has multiple `ui`s in the `uis` folder, a common router object in `routes.R`. Please note that the example code referenced above is quite outdated and would need some dev love, but the idea should work nevertheless.
